lib/Tumblr/StreamBuilder/StreamFilters/StreamFilter.php

```diff
@@ -133,7 +134,7 @@ final public function filter(
     /**
      * The filter concrete logic implementation.
      * @param StreamElement[] $elements The candidate StreamElement instances to filter.
-     * @param StreamFilterState $state The filter state passed from previous filter operations.
+     * @param StreamFilterState|null $state The filter state passed from previous filter operations.
```

Comment:
i think the argument itself needs to be updated to `?StreamFilterState` for this to be true....